core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -214,6 +214,23 @@ ManifestFile writeManifest(Long snapshotId, ManifestEntry<?>... entries) throws
     return writer.toManifestFile();
   }
 
+  @SuppressWarnings("checkstyle:HiddenField")
```

Comment:
Nit: could you please use another name for the `formatVersion`, for example `newFormatVersion` ,  so that we don't need the extra checkstyle suppressing ?

Reply:
After apply the suggestion [here](https://github.com/apache/iceberg/pull/2271/files#r586151267),  we could just use the private `formatVersion` memember.